Build/build.bat

```diff
@@ -3,3 +3,5 @@ copy /Y ..\Source\ExcelDna\Release64\ExcelDna64.xll ..\Distribution\
 copy /Y ..\Source\ExcelDna.Integration\bin\Release\ExcelDna.Integration.dll ..\Distribution\
 copy /Y ..\Source\ExcelDnaPack\bin\Release\ExcelDnaPack.exe ..\Distribution\
 copy /Y ..\Source\ExcelDnaPack\bin\Release\ExcelDnaPack.exe.config ..\Distribution\
+copy /Y ..\Source\ExcelDna.AddIn.Tasks\bin\Release\ExcelDna.AddIn.Tasks.dll ..\Package\ExcelDna.AddIn\tools\
+copy /Y ..\Source\ExcelDna.AddIn.Tasks\bin\Release\ExcelDna.AddIn.Tasks.pdb ..\Package\ExcelDna.AddIn\tools\
```

Comment:
It's a common practice to include .pdb files in NuGet packages, so that we can get detailed exception information, in the case of errors.

ExcelDna.Integration.pdb is not required for this PR to work, but I would recommend it to be included in future packages for the same reason.

Most people nowadays don't commit nuget packages to source control, and use package restore, so saving one or two megabytes in the package size makes little difference.